app/graphql/types/customer/customer.gql

```diff
@@ -28,7 +28,7 @@ type CustomerAuthenticationQuestions {
 }
 
 type Customer {
-  referenceNumber: ID!
```

Comment:
Is there a reason we changed this? We may encounter other customer ids than CRN so maybe best to leave this as is?

Reply:
so actually ID is not the crn, there is a crn inside the customer info